user-app/main.py

```diff
@@ -0,0 +1,17 @@
+if __name__ == "__main__":
+    # load data from csv
+    from scripts.readfiles import readCsv
+    df = readCsv('tripadivsordata',nrows=100)
+
+    # run sentiment analysis 
+    from scripts.sentiment import sentAnalysisApp
+    sent_app = sentAnalysisApp(df,'review')
```

Comment:
Calling the explicit column name is not very handy: maybe by default it will always use the second (or first column) in a csv file?

Reply:
nee je moet dit ook niet zo zien, de main file is even tijdelijk todat we daadwerkelijk een interface hebben

Reply:
Ah oke, ik denk dat ik dat nog niet goed begrijp of voor me kan zien :)